traffic_signs_detection/traffic_signs_detection/detectorPolygons.cpp

```diff
@@ -0,0 +1,15 @@
+#include <iostream>
+#include "detectorPolygons.h"
+
+
+std::vector<cv::Vec3f> detectorPolygons::detectCircle(cv::Mat frame)
```

Comment:
Объекты размером больше 8 байт в C++ надо по константной ссылке передавать:
```suggestion
std::vector<cv::Vec3f> detectorPolygons::detectCircle(const cv::Mat &frame)
```
Иначе они копируются при каждом вызове, это не очень с точки зрения скорости работы.